packages/storage/src/platform/browser/connection.ts

```diff
@@ -22,6 +22,9 @@ import {
 } from '../../implementation/connection';
 import { internalError } from '../../implementation/error';
 
+/** An override for the text-based Connection. Used in tests. */
```

Comment:
Should Connection be backticked as a literal?  Looks like there are other instances of the same in other files in this PR.